Anature/src/application/anatures/movesets/MoveSet.java

```diff
@@ -190,6 +190,31 @@ public boolean setMovePoints(int moveNumber, int movePoints)
 		}
 	}
 
+	public int getMoveIndex(IMove move)
+	{
+		if(mMove1.getName().compareTo(move.getName()) == 0)
+		{
+			return 1;
+		}
+
+		else if(mMove2.getName().compareTo(move.getName()) == 0)
+		{
+			return 2;
+		}
+
+		else if(mMove3.getName().compareTo(move.getName()) == 0)
+		{
+			return 3;
+		}
+
+		else if(mMove4.getName().compareTo(move.getName()) == 0)
+		{
+			return 4;
+		}
+
+		return -1;
```

Comment:
I believe you can just use `mMove#.equals(move)` because there **should** only ever be one instance of any move.

Either way, it works! :shipit:

Reply:
Yeah I thought about that, but figured I'd rather be safe than sorry.